Podman/podman-compose.ci.yml

```
@@ -0,0 +1,44 @@
+      - TZ=${TZ:-UTC}
+      - BIRDNET_UID=${BIRDNET_UID:-1000}
+      - BIRDNET_GID=${BIRDNET_GID:-1000}
+      # CI-specific: Disable audio processing since no hardware available
+      - BIRDNET_REALTIME_AUDIO_ENABLED=false
+    volumes:
+      - ./config:/config
+      - ./data:/data
+    # Mount HLS stream segment files directory as tmpfs (RAM disk)
+    tmpfs:
+      - /config/hls:exec,size=50M,uid=${BIRDNET_UID:-1000},gid=${BIRDNET_GID:-1000},mode=0755
+    # NOTE: Audio device removed for CI environment
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)